### DISCUSSION TITLE
#mesh foot displacement

### DISCUSSION THREAD

**Discussion by JoeGitcpp** [Sixze/ALS-Refactored#285] [Q&A] [answered]
- bing refresh IK Bone Name
- Use Als physics Asset
// Problem are Comefrome my  mesh ,i just want know why 
![Debug2](https://user-images.githubusercontent.com/104304808/231912166-bf85bd57-5911-4589-94ef-650c47ca7fe2.png)

**Comment by JoeGitcpp** [comment]
But If I disable Control Rig,problem is gone
![OK](https://user-images.githubusercontent.com/104304808/231913464-1595e9f5-aa72-4b33-bef3-eee395bfb3fc.png)

**Reply by assmikDuDuDuJ** [reply]
iK bones are there ?

**Reply by JoeGitcpp** [reply]
Yes im not touch Anything

**Reply by assmikDuDuDuJ** [reply, marked as answer]
It happens by itself, for example, when I import the animation/mesh/skeleton.
IK bones can rotate\displace for some reason.
I had a similar problem when the ik bones broke.
or  problem with the variable FeetState.Right.IkRotation\FeetState.Left.IkRotation
If I'm not mistaken.
But I would look towards the mеsh bones if the C++ code wasn't edited.
